### PR TITLE
OPENEUROPA-636: Reduce patches on OpenEuropa Multilingual

### PR DESCRIPTION
**Pull request by ademarco**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,8 +100,8 @@ The setup procedure described above can be sensitively simplified by using Docke
 
 Requirements:
 
-- [Docker][8]
-- [Docker-compose][9]
+- [Docker][6]
+- [Docker-compose][7]
```
Comment (voidtek) [inline README.md]: This two links are removed in the list. see down.

**Comment by netlooker** [comment]
Something went wrong. Fresh build&install gives following error:
```
 ---- -------------- -------- ---------- -------------------------------------- 
  ID   Date           Type     Severity   Message                               
 ---- -------------- -------- ---------- -------------------------------------- 
  31   14/Jun 16:47   php      Error      Twig_Error_Loader: Template           
                                          "@ecl/ec-component-button" is not     
                                          defined in                            
                                          "themes/contrib/oe_theme/templates/f  
                                          orm/input--submit.html.twig" at line  
                                          14. in                                
                                          Twig_Loader_Chain->getCacheKey()      
                                          (line 129 of                          
                                          /var/www/html/vendor/twig/twig/lib/T  
                                          wig/Loader/Chain.php).
```

**Comment by aritomelo** [comment]
I have installed a brand new instance in C9 and it works fine.

**Comment by netlooker** [comment]
The error that I have attached in the previous commit was caused by some unknown/unfound issue on AWS Cloud9 environment. After deleting and creating a new AWS environment repeating all of the steps which are required to perform review went fine and without any errors.

**Pull request merged**